types/k6/experimental/browser.d.ts

```diff
@@ -1045,12 +1050,163 @@ export interface ElementHandle extends JSHandle {
      */
     boundingBox(): Rect;
 
+    /**
+     * Checks the checkbox element.
+     * @param options The options to use.
+     */
+    check(options?: ElementClickOptions & StrictnessOptions): void;
+
+    /**
+     * Clicks the element.
+     * @param options The options to use.
+     * @returns A promise that resolves when the element is clicked.
+     */
+    click(
+        options?: {
```

Comment:
From what I can tell, these options and the options for `dblclick` differ only by `clickCount`. In order to reduce the duplication, mainly in the comments, I'd extract an interface and use inheritance:

```
interface CommonClickOptions {
  ...
}

interface ClickOptions extends CommonClickOptions  {
  /**
   * The number of times the action is performed. Defaults to `1`.
   */
  clickCount?: number
}
```

Reply:
I do like this idea. My main concern is that if we introduce an interface we can't then change it too easily in the future if we need to since it will cause breaking changes to users scripts (I think).

I'd prefer to keep duplications for now, and tidy the duplications all in one go once we're at a stage to be able to do this.